target/linux/ath79/dts/qca9531_glinet_gl-x300b.dtsi

```diff
@@ -0,0 +1,131 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
Move everything from the dtsi to the dts, as i don't see another consumer.

Reply:
There are two [different versions](https://www.gl-inet.com/products/gl-x300b/) (scroll down to model comparison) of this router, the -RS485 and the -IOT. The IOT version includes a Bluetooth or Zigbee module. This PR only supports the hardware in the RS485, it should work on the -IOT but without support for the module. If someone were to develop support for the IOT version they would be able to use the DTSI.

Should I merge them as you requested, or leave them separate allowing someone to implement support for the IOT version more easily in the future?

Reply:
As discussed on IRC I will merge the dts and dtsi files.